components/weather_data/WeatherStationList.tsx

```diff
@@ -37,15 +53,84 @@ export const WeatherStationList: React.FC<Props> = ({center_id, requestedTime})
   return <StationList center_id={center_id} token={metadata.widget_config.stations?.token} requestedTime={requestedTime} />;
 };
 
+const EARTH_RADIUS = 6378.1; // in kilometers
```

Comment:
I find it helpful to append the units to the variable name when there's potential for confusion
```suggestion
const EARTH_RADIUS_KM = 6378.1;
```
(you did this yourself in `radiusForExtent` 😄)